### DISCUSSION TITLE
Which US state is nicknamed the "Pine Tree State"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#266] [Galaxy Trivia] [answered]
**Hint:** northeasternmost

**Comment by tomkuttler** [comment, marked as answer]
Maine

**Reply by ParanoidUser** [reply]
You nailed it! Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
